platforms/documentation/docs/src/docs/userguide/best-practices/bp_general.adoc

```diff
@@ -57,3 +57,57 @@ include::sample[dir="snippets/bestPractices/usePluginsBlock/groovy/do-this",file
 
 === Tags
 `#structuring-builds`
+
+[[use_convention_plugins]]
+== Use Convention Plugins for Common Build Logic
+
+You should use convention plugins to encapsulate and reuse common build logic across multiple build files.
+
+=== Explanation
+
+Convention plugins help you avoid duplicating build logic across multiple build files by moving shared configuration into a reusable plugin. This approach:
```

Comment:
```suggestion
Instead of duplicating configuration across multiple `build.gradle.kts` files, you can move common logic into a reusable plugin. This approach offers several benefits:
```